mne/minimum_norm/tests/test_resolution_matrix.py

```diff
@@ -8,13 +8,15 @@
 import os.path as op
 import numpy as np
 from numpy.testing import (assert_equal, assert_array_almost_equal,
-                           assert_array_equal)
+                           assert_array_equal, assert_allclose)
 
 import mne
 from mne.datasets import testing
 from mne.minimum_norm.resolution_matrix import (make_inverse_resolution_matrix,
                                                 get_cross_talk,
-                                                get_point_spread)
+                                                get_point_spread,
+                                                _vertices_for_get_psf_ctf)
+from mne.utils import run_tests_if_main
```

Comment:
Please don't add this back in, eventually it should go away (it's not useful really now that we use `pytest`)

Reply:
I guess I should remove it from test_resolution_metrics.py as well?

Reply:
Yes feel free, I remove them as I come across them when making changes to a given file